### PR TITLE
Added recipe for glue-astronomy

### PR DESCRIPTION
**Pull request by astrofrog**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] <s>If static libraries are linked in, the license of the static library is packaged.</s>
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by astrofrog-conda-forge** [comment]
I confirm that I am happy to maintain this

**Comment by astrofrog** [comment]
@conda-forge/help-python - this is ready for review!

File: recipes/glue-astronomy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1" %}
+
+package:
+  name: glue-astronomy
+  version: {{ version }}
+
+source:
+  fn: glue-astronomy-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:8]: you can drop the `fn`

File: recipes/glue-astronomy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://pypi.io/packages/source/g/glue-astronomy/glue-astronomy-{{ version }}.tar.gz
+  sha256: c31f8b1c95367a09c85e6719d4313fdb7a9acaf68d9abb96c6af68ebcdcde02d
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:14]: `script: "{{ PYTHON }} -m pip install . -vv"`

File: recipes/glue-astronomy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+
+requirements:
+
+  build:
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:19]: `build` -> `host`

File: recipes/glue-astronomy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - pip
+    - setuptools >=30.3
+    - setuptools_scm
+
+  run:
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:25]: This package looks like it uses `pkg_resources` at runtime, so should it have `setuptools` under `run`?

File: recipes/glue-astronomy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:20]: `python >=3.6`

File: recipes/glue-astronomy/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+    - setuptools >=30.3
+    - setuptools_scm
+
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/glue-astronomy/meta.yaml:26]: `python >=3.6`

File: recipes/glue-astronomy/meta.yaml (hunk 1 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml]: Done

File: recipes/glue-astronomy/meta.yaml (hunk 2 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml]: Done

File: recipes/glue-astronomy/meta.yaml (hunk 3 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml]: Done

File: recipes/glue-astronomy/meta.yaml (hunk 4 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml:24]: Done

File: recipes/glue-astronomy/meta.yaml (hunk 5 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml]: Done

File: recipes/glue-astronomy/meta.yaml (hunk 6 above)
Comment (astrofrog) [reply recipes/glue-astronomy/meta.yaml]: Done

**Comment by astrofrog** [comment]
@synapticarbors - thanks for your review! I've implemented your comments.

**Pull request merged**